### ISSUE TITLE
FailedDownloadException on launch

### ISSUE THREAD

**Issue by rivella50** [tschulte/gradle-jnlp-plugin#21]
Hi,
i just tried your plugin, where signing and copying to the right place seems to work fine.
But when invoking launch.jnlp i get the mentioned exception for the first library in the launch file:
```
com.sun.deploy.net.FailedDownloadException: Unable to load resource: file:/Users/xyz/Development/java/project/mainapp/build/jnlp/lib/joda-time__V2.7.jar
```
The file can be found in lib dir with the name joda-time__V2.7.jar.pack.gz
Also when i change the href to 
```
<jar href='lib/joda-time__V2.7.jar.pack.gz' />
```
i get the same error.
Is there something else needed to start the application ?
I'm working on OS X.

Thank you!

EDIT: Ok this is not a problem, the app has to be started with an URL and cannot be opened from console!

**Comment by tschulte** [comment]
Yes, you are right. I recently changed the default to use pack200, see #19. You can configure it to not use pack200 though. 

But if you want to launch from a file share, you should consider using a batch/bash file or launchable jar. The output of the plugin is meant to be deployed in a HTTP server like Apache. But once #19 is done, creating a war file with servlet (and jardiff) will be very easy. Once this is done I plan on doing some documentation.

**Comment by rivella50** [comment]
Thank you for the explanation.
I got it all working and launch from a file share is actually not needed for my purposes.
